src/controllers/todo/todo.controller.ts

```diff
@@ -3,81 +3,79 @@ import { Document } from "mongoose";
 import TodoItem from "../../interfaces/todo";
 import { TodoModel } from "./../../models/todo/todo.model";
 
-class TodoController {
-  public create(req: Request, res: Response): void {
-    const todoItem = new TodoModel(req.body);
-    todoItem.save()
-    .then((todo: Document): void => {
-      res.status(201);
-      res.json({ status: true, message: todo});
-    })
-    .catch((error: Error): void => {
-      res.status(500);
-      res.json({ status: false, error });
-    });
-  }
+function create(req: Request, res: Response): void {
```

Comment:
Good work here. But you can make `create` an async function as well - not that you should

Reply:
Great idea, I've done this in [this pull request](https://github.com/Lemmah/diligent-time-api/pull/8).